tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts

```diff
@@ -58,7 +58,6 @@ export const DataLoaderBehavior = {
 
     dataLoading: {
       type: Boolean,
-      readOnly: true,
```

Comment:
Does this have to be non-readOnly in order to use set()?  It's still logically readonly though, right?

Reply:
Ah, you were correct! I always forget that you ned to use a special setter when setting a value to a readOnly property. I changed other codes to use `this._setDataLoading` and kept this readOnly. Thanks!